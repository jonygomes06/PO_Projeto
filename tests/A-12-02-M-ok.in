1
faltoso
7
1
1
2
1
3
2
2
2
2
2
2
2
2
12
2
1
1
3
3
2
3
3
0
0
