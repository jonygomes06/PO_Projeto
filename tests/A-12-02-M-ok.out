Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Ficheiro a abrir: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 2 deve ser devolvida até ao dia 29.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 2 deve ser devolvida até ao dia 29.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 2 não foi requisitada pelo utente 2.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 2 não foi requisitada pelo utente 2.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: O utente 12 não existe.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 3 deve ser devolvida até ao dia 36.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Opção inválida!
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 3 não foi requisitada pelo utente 3.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 